Adds security warning on secrets in containers (#170)
carpentries-incubator/docker-introduction: _episodes/advanced-containers.md

run `docker image ls` you'll see the size of each image all the way on the right of
the screen. The bigger your image becomes, the harder it will be to easily download.

> ## Security warning
> Login credentials including passwords, tokens, secure access tokens or other secrets

> > must never be stored in a container. If secrets are stored, they will be found and
> > exploited when made public - either in the near or distant future.

Reviewer: ```suggestion
> must never be stored in a container. If secrets are stored, they are at high risk to be found and
> exploited when made public.
```
Small suggestion for a language edit but I'm okay with either.

Author: Done. Thanks for the suggestion.
